docs/setup.md

```diff
@@ -0,0 +1,35 @@
+In order to use the react eslint plugin do the following...
+
+1. At the root of your project type <code>npm install eslint-plugin-react --save-dev</code>
```

Comment:
if you want to truly cover things for newbies, you may want to include `npm init` and `eslint` itself.

Reply:
I think we can assume these as necessary knowledge to have even got this doc